### PR TITLE
Docs/UserGuide: Update Keith's features

### PR DESCRIPTION
**Pull request by KeithSOC**
Update features to be implemented by Keith

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -67,11 +67,56 @@ Examples:
...
+=== Listing all persons: `list`
 
 Shows a list of all persons in the address book. +
 Format: `list`
 
+=== Listing all favorite persons: `listfav`
+
+Shows a list of all favorite contacts in the address book. +
+Format: `listfav`
+
```
Comment (marvinchin) [inline docs/UserGuide.adoc]: can we extend the list command instead? for instance using `list fav` instead of adding a new command entirely

Comment (KeithSOC) [reply docs/UserGuide.adoc]: Agreed! This is a better idea 👍 I'll update it ✅

Comment (marvinchin) [reply docs/UserGuide.adoc]: Has this been updated? @KeithSOC

Comment (KeithSOC) [reply docs/UserGuide.adoc]: Updated ✔

**Pull request merged**